### PR TITLE
[AWIBOF-7458] Revise help message for SETSYSTEMPROPERTY and CREATEARR…

### PR DESCRIPTION
**Pull request by mjlee34**
…AY commands

Revise the help message for SETSYSTEMPROERPTY and CREATEARRAY commands.

- Remove the description from SETSYSTEMPROPERTY command describing perf-impact flag is provided as an
experimental feature.
- Change [highest] and [lowest] to [high] and [low].
- Add the description to CREATEARRAY command describing RAID6 is provided as an experimental feature.

Signed-off-by: mjlee34 <jun20.lee@samsung.com>

### CODE REVIEW

**Review by ringopow — APPROVED** [review]
수고하셨습니다.

File: tool/cli/cmd/systemcmds/set_system_property.go [hunk 1]
```diff
@@ -18,14 +18,12 @@ var SetSystemPropCmd = &cobra.Command{
 	Short: "Set the property of PoseidonOS.",
 	Long: `
 Set the property of PoseidonOS.
-(Note: this command is not officially supported yet.
- It might be  possible this command cause an error.)
 
 Syntax:
-	poseidonos-cli system set-property [--rebuild-impact (highest | medium | lowest)]
+	poseidonos-cli system set-property [--rebuild-impact (high | medium | low)]
```
Comment (minjoon-ahn) [inline tool/cli/cmd/systemcmds/set_system_property.go:23]: 서버는 바꿀필요없을까요?

**Review by minjoon-ahn — DISMISSED** [review]

File: tool/cli/cmd/systemcmds/set_system_property.go (hunk 1 above)
Comment (mjlee34) [reply tool/cli/cmd/systemcmds/set_system_property.go:23]: 제가 확인해보니 서버는 이미 high low 로 되어있는 것 같습니다.

**Review by minjoon-ahn — APPROVED** [review]
LG2M

**Review by junghyun-34 — APPROVED** [review]
LG2M

**Pull request merged**